File: C/Listas/Lista 01/ex13.c
//Leia o numerador e o denominador de uma fração e transforme-o em um número decimal.
#include <stdio.h>
#include <stdlib.h>

int main(){
    float num, denom;
    printf("Insira o valor do numerador: ");
    scanf("%f", &num);
    printf("Insira o valor do denominador: ");
    scanf("%f", &denom);

    printf("A sua fração é %.0f/%.0f",num, denom);
    printf("\nTransformando-a em decimal: %.2f", num/denom);
    return 0;
}

File: Java/extra/Lista 01/src/ex13.java
// Leia o numerador e o denominador de uma fração e transforme-o em um número decimal

import java.util.Scanner;

public class ex13 {
    static double decimal(int a, int b) {
        double x = (double) a / (double) b;
        return x;
    }

    public static void main(String[] args) {
        int num, denom;
        System.out.println("Informe o valor do numerador: ");
        try (Scanner leitor = new Scanner(System.in)) {
            num = leitor.nextInt();
            System.out.println("\nDigite o valor do denominador: ");
            denom = leitor.nextInt();
        }
        System.out.format("\nA fração %d/%d em forma decimal é %.2f", num, denom, decimal(num, denom));
    }
}
